hardware/action_examples/hls_search/hls_search.cpp

```diff
@@ -745,7 +751,7 @@ adding_padding_to_256bytes_to ensure test  ok
      */
 
     // read data for text
-    fp = fopen("../../../../snap_search123.txt", "r");
```

Comment:
brrrr. so many .. ... .. . I do not like this, can we make this simpler or differently? Is that the test bench code?